### ISSUE TITLE
Latest multi-dev breaks windows

### ISSUE THREAD

**Issue by scottwsides** [conda/constructor#544]
### Checklist

- [X] I added a descriptive title
- [X] I searched open reports and couldn't find a duplicate

### What happened?

The following works for windows installer build. If the git checkout line is removed.... it definitely breaks. 

```
git clone https://github.com/conda/constructor.git cs-constructor

echo -----------
cd  cs-constructor
git checkout 8426b02ca72d277fa7f538577445e8f5da0bd573 .  <-----------------------------
rem python setup.py install
python -m pip install -e . --no-deps

```

**Comment by jaimergp** [comment]
These are the changes as of now

https://github.com/conda/constructor/compare/8426b02..31e1947

There are changes in how the NSIS script is written, but the CI does pass with a fresh installation, so I am wondering if the error is specific to the way is set up on your machine.

Can you provide the input file you are using for this installer, plus the full logs containing the error?

**Comment by scottwsides** [comment]
It seems very latest construtor (one more commit than when I saw bug) no longer has this problem. I'm still double-checking and will let you know... but I think you can de-prioritize.

**Comment by jaimergp** [comment]
I just ran into it while debugging as well so... I'll keep an eye on this :)

**Comment by jaimergp** [comment]
These are the logs:

```
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\Lib
Extract: _nsis.py
Extract: _system_path.py
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\conda-meta
Extract: history
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565
Extract: _conda.exe
Extract: pre_uninstall.bat
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs
Extract: urls
Extract: urls.txt
Extract: post_install.bat
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs\cache
Extract: 0c905de7.json
Extract: d02f2982.json
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs\bzip2-1.0.8-he774522_0\info
Extract: repodata_record.json
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs\ca-certificates-2022.07.19-haa95532_0\info
Extract: repodata_record.json
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs\certifi-2022.6.15-py310haa95532_0\info
Extract: repodata_record.json
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs\libffi-3.4.2-hd77b12b_4\info
Extract: repodata_record.json
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs\openssl-1.1.1q-h2bbff1b_0\info
Extract: repodata_record.json
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs\pip-22.1.2-py310haa95532_0\info
Extract: repodata_record.json
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs\python-3.10.4-hbb2ffb3_0\info
Extract: repodata_record.json
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs\setuptools-61.2.0-py310haa95532_0\info
Extract: repodata_record.json
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs\sqlite-3.39.2-h2bbff1b_0\info
Extract: repodata_record.json
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs\tk-8.6.12-h2bbff1b_0\info
Extract: repodata_record.json
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs\tzdata-2022a-hda174b7_0\info
Extract: repodata_record.json
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs\vc-14.2-h21ff451_1\info
Extract: repodata_record.json
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs\vs2015_runtime-14.27.29016-h5e58377_2\info
Extract: repodata_record.json
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs\wheel-0.37.1-pyhd3eb1b0_0\info
Extract: repodata_record.json
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs\wincertstore-0.2-py310haa95532_2\info
Extract: repodata_record.json
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs\xz-5.2.5-h8cc25b3_1\info
Extract: repodata_record.json
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs\zlib-1.2.12-h8cc25b3_2\info
Extract: repodata_record.json
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs
Extract: python-3.10.4-hbb2ffb3_0.tar.bz2
Extract: ca-certificates-2022.07.19-haa95532_0.conda
Extract: tzdata-2022a-hda174b7_0.conda
Extract: vs2015_runtime-14.27.29016-h5e58377_2.conda
Extract: vc-14.2-h21ff451_1.conda
Extract: bzip2-1.0.8-he774522_0.conda
Extract: libffi-3.4.2-hd77b12b_4.conda
Extract: openssl-1.1.1q-h2bbff1b_0.conda
Extract: sqlite-3.39.2-h2bbff1b_0.conda
Extract: tk-8.6.12-h2bbff1b_0.conda
Extract: xz-5.2.5-h8cc25b3_1.conda
Extract: zlib-1.2.12-h8cc25b3_2.conda
Extract: certifi-2022.6.15-py310haa95532_0.conda
Extract: wheel-0.37.1-pyhd3eb1b0_0.conda
Extract: wincertstore-0.2-py310haa95532_2.conda
Extract: setuptools-61.2.0-py310haa95532_0.conda
Extract: pip-22.1.2-py310haa95532_0.conda
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs
Extract: env.txt
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\conda-meta
Collecting package metadata (current_repodata.json): ...working... done
Solving environment: ...working... failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - wincertstore==0.2=py310haa95532_2
  - xz==5.2.5=h8cc25b3_1
  - python==3.10.4=hbb2ffb3_0
  - tk==8.6.12=h2bbff1b_0
  - libffi==3.4.2=hd77b12b_4
  - vc==14.2=h21ff451_1
  - zlib==1.2.12=h8cc25b3_2
  - sqlite==3.39.2=h2bbff1b_0
  - ca-certificates==2022.07.19=haa95532_0
  - wheel==0.37.1=pyhd3eb1b0_0
  - vs2015_runtime==14.27.29016=h5e58377_2
  - certifi==2022.6.15=py310haa95532_0
  - pip==22.1.2=py310haa95532_0
  - openssl==1.1.1q=h2bbff1b_0
  - bzip2==1.0.8=he774522_0
  - tzdata==2022a=hda174b7_0
  - setuptools==61.2.0=py310haa95532_0

Current channels:

  - http://repo.anaconda.com/pkgs/main/win-64
  - http://repo.anaconda.com/pkgs/main/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.


Output folder: D:\Users\jrodriguez\AppData\Local\noconda565
Delete file: D:\Users\jrodriguez\AppData\Local\noconda565\pkgs\env.txt
Output folder: D:\Users\jrodriguez\AppData\Local\noconda565\conda-meta
Extract: history
Creating NoConda menus...
Running mkdirs...
Clearing package cache...
Remove all contents from the following package caches?
  - D:\Users\jrodriguez\AppData\Local\noconda565\pkgs
Created uninstaller: D:\Users\jrodriguez\AppData\Local\noconda565\Uninstall-NoConda.exe
Completed
```

**Comment by jaimergp** [comment]
Found the issue. Fix coming soon in #553
